pandas/tests/indexing/test_chaining_and_caching.py

```diff
@@ -421,19 +421,6 @@ def test_detect_chained_assignment_warnings_errors(self):
             with pytest.raises(com.SettingWithCopyError, match=msg):
                 df.loc[0]["A"] = 111
 
-    def test_detect_chained_assignment_warnings_filter_and_dupe_cols(self):
```

Comment:
is this test redundant or just the expected behavior changed?

Reply:
Behavior changed, this should be more intuitive now